### PR TITLE
feat(blog): add Camunda Modeler v4.0.0 blog post

### PR DESCRIPTION
**Pull request by philippfromme**
Related to https://github.com/camunda/camunda-modeler/issues/1802

### CODE REVIEW

File: content/post/2020/05/camunda-modeler-4.0.0-released.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
++++
+author = "Philipp Fromme"
+categories = ["Modeling"]
+tags = ["BPMN 2.0", "Tooling", "Release Note", "Camunda Modeler"]
+date = "2020-05-27T15:00:00+02:00"
```
Comment (nikku) [inline content/post/2020/05/camunda-modeler-4.0.0-released.md:5]: Can you mark this one as a draft so it won't be accidentally published?

Comment (philippfromme) [reply content/post/2020/05/camunda-modeler-4.0.0-released.md:5]: 👍

File: content/post/2020/05/camunda-modeler-4.0.0-released.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+Following the top to bottom layout, created connections will connect newly created elements to already existing ones. Furthermore, the layout of information requirements has been adjusted to support the tree structure resulting from the top to bottom approach.
+
+## ...Without Enforcing Them
+
+Even though we encourage you to layout your decision requirement diagrams top to bottom, you can still create diagrams any way you want. If you prefer modeling left-to-right, simply drag the elements from the context pad instead of clicking.
+
+## Overhauled Deployments from the Camunda Modeler
+
+Deploying diagrams from the Camunda Modeler has been an integral feature since its introduction. With v4.0.0, we have completely overhauled the UI of deployments. This marks the first step in streamlining the Camunda Modeler UI.
```
Comment (oguzeroglu) [inline content/post/2020/05/camunda-modeler-4.0.0-released.md:43]: `UI of deployments` -> `UX of deployments` or `UI of the deployment dialog`?

File: content/post/2020/05/camunda-modeler-4.0.0-released.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Last but not least
+
+[As a part of the Camunda rebranding](https://blog.camunda.com/post/2020/01/something-bold-something-new-fresh-look/), we say goodbye to our old Camunda Modeler icon and introduce a fresh new icon. You may see this icon in your taskbar, app launch menu, etc.
+
+{{< figure class="no-border teaser" src="4-icon.png" width="100px" alt="New Camunda Modeler icon" caption="New Camunda Modeler icon" >}}
+
+## Looking Forward
+
+Read the [changelog](https://github.com/camunda/camunda-modeler/blob/master/CHANGELOG.md) to learn about all features, bug fixes, and numerous additional improvements that made it into this release. [Download the new modeler](https://camunda.com/download/modeler/), give it a try, and, most importantly, give us feedback.
```
Comment (oguzeroglu) [inline content/post/2020/05/camunda-modeler-4.0.0-released.md:63]: learn about all _the_ features

File: content/post/2020/05/camunda-modeler-4.0.0-released.md (hunk 2 above)
Comment (philippfromme) [reply content/post/2020/05/camunda-modeler-4.0.0-released.md:43]: 👍

File: content/post/2020/05/camunda-modeler-4.0.0-released.md (hunk 3 above)
Comment (philippfromme) [reply content/post/2020/05/camunda-modeler-4.0.0-released.md:63]: 👍

File: content/post/2020/05/camunda-modeler-4.0.0-released.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+DMN 1.3 is the upcoming version of the DMN standard. Compared to DMN 1.1 version 1.3 of the standard ships with a number of improvements, most importantly, a diagram interchange that allows graphical information to be exchanged across vendors.
+The app now exports DMN 1.3 and uses the standardized diagram interchange when doing so. You can open older DMN 1.1 or DMN 1.2 files; however, those will be migrated to DMN 1.3 on save.
+
+## Camunda BPM Compatibility with DMN 1.3
+
+DMN 1.3 diagrams created with Camunda Modeler v4.0.0 are compatible with the following versions of Camunda BPM: 7.13.0, 7.12.4, 7.11.11, 7.10.17 and above. Make sure to update your Camunda engine accordingly if you are using DMN 1.3.
+
+## Simplified DMN Modeling
+
+Besides adopting the latest DMN standard, we are continuously investing in our DMN modeler to make it an even better tool for decision modeling. With the introduction of [automatic layout in the BPMN modeler](https://blog.camunda.com/post/2018/01/camunda-modeler-1120-alpha-2-released/), creating well laid out diagrams became both easier and faster. This release finally brings this feature to the DRD editor of the Camunda Modeler.
```
Comment (volkergersabeck) [inline content/post/2020/05/camunda-modeler-4.0.0-released.md:27]: I believe, we're linking to the wrong blog post in here

Comment (philippfromme) [reply content/post/2020/05/camunda-modeler-4.0.0-released.md:27]: Do you mean https://blog.camunda.com/post/2018/01/camunda-modeler-1120-alpha-2-released/? It's actually the right blog post.

image

Might be a bit hard to spot.

Comment (barmac) [reply content/post/2020/05/camunda-modeler-4.0.0-released.md:27]: Yeah, it's not even possible to link to that bullet point. The closest one can get is [link to the section](https://blog.camunda.com/post/2018/01/camunda-modeler-1120-alpha-2-released#other-improvements).

Comment (philippfromme) [reply content/post/2020/05/camunda-modeler-4.0.0-released.md:27]: Alright, I changed the link. Thx @barmac 👍

**Comment by oguzeroglu** [comment]
Why is this still open?

**Comment by nikku** [comment]
Guess left over. It is merged outside the GitHub merge flow:

image

So closing it!

**Pull request closed without merging**